src/libm-tester/iutsimd.c

```diff
@@ -416,12 +416,7 @@ int do_test(int argc, char **argv) {
     func_d_d("exp10", xexp10);
     func_d_d("expm1", xexpm1);
     func_d_d("log10", xlog10);
-    func_d_d("log1p", xlog1p);
```

Comment:
Could you please explain why you are removing these lines?

Reply:
These lines were accidentally duplicated in the previous code.
The same tests was executed twice, which was just waste of time.